packages/calypso-build/README.md

```diff
@@ -113,8 +113,8 @@ To extend the default behavior provided by `@automattic/calypso-build`, you can
 ```js
 module.exports = {
 	presets: [
-		require( '@automattic/calypso-build/babel/default' ),
 		require( '@automattic/calypso-build/babel/wordpress-element' ),
+		require( '@automattic/calypso-build/babel/default' ),
```

Comment:
Should these be wrapped in `require`?

Reply:
Works with and without `require()`. In some cases, the `require()` provides some safety against accidentally trying to use a preset that doesn't exist (and e.g. ESLint can complain that the package hasn't been added to the project's `package.json`). But yeah, not strictly needed here, and arguably a bit against convention. I'll drop the `require()`s.

Reply:
b24083cf54